example/image-classification/test_score.py

```diff
@@ -0,0 +1,46 @@
+"""
+test pretrained models
+"""
+import argparse
+import mxnet as mx
+from common import find_mxnet, modelzoo
+from common.util import download_file
+from score import score
+
+def download_data():
+    download_file('http://data.mxnet.io/data/val-5k-256.rec', 'data/val-5k-256.rec')
+
+def test_imagenet1k_resnet(args):
+    models = ['imagenet1k-resnet-34',
+              'imagenet1k-resnet-50',
+              'imagenet1k-resnet-101',
+              'imagenet1k-resnet-152']
+    accs = [.72, .75, .765, .76]
+    for (m, g) in zip(models, accs):
+        acc = mx.metric.create('acc')
+        (speed,) = score(model=m, data_val='data/val-5k-256.rec',
+                         rgb_mean='0,0,0', metrics=acc, **vars(args))
```

Comment:
@piiswrong actually we can do this, but i think we'd better not do that and add one parameters like ```rgb_mean``` . because no need mean value is more convenient than the value like (123.68,116.779,103.939) and this is original considered for resnet training on purpose.
another, in some training project(such as face-related model ), we usually use mean=[127.5, 127.5, 127.5], and this is also a different mean value. or on some other field, the mean value may even own a big difference than 127.  
so we can give write corresponding code for some famous model, and just mention it in the readme file.